src/main/java/org/sagebionetworks/web/client/presenter/EntityPresenter.java

```diff
@@ -364,35 +361,35 @@ public void onError(Throwable caught) {
   }
 
   public void show403() {
-    if (entityId != null) {
-      synAlert.show403(entityId, versionNumber);
-    }
+    synAlert.show403(entityId, versionNumber);
```

Comment:
does this work ok if the entity is nonsense, like if the place is `Synapse:notASynId`? Just wondering if the null check is relevant to parsing the place token

Reply:
Works with an invalid synapse ID place token (we do attempt to get the DOI information, which results in a 400, but this does not impact the UI).